client/imports/account/account.html

```diff
@@ -0,0 +1,54 @@
+<ul id = "toolbar">
+  <li><a class="active" href="./home">Home</a></li>
+  <li><a href="./account">Account</a></li>
+  <li><a href="./coursepage">Courses</a></li>
+</ul>
+
+<style>
+#header {
+    background-color:black;
+    color:white;
+    text-align:center;
+    padding:5px;
+}
+#nav {
+    line-height:30px;
+    background-color:#eeeeee;
+    height:300px;
+    width:100px;
+    float:left;
+    padding:5px;
+}
+#section {
+    width:350px;
+    float:left;
+    padding:10px;
+}
+#footer {
+    background-color:black;
+    color:white;
+    clear:both;
+    text-align:center;
+    padding:5px;
+}
+</style>
+
+<div id="header">
+<h1>15131:Great Practical Ideas in CS</h1>
+</div>
+
+<div id="nav">
```

Comment:
This is really problematic-

1) The tasks need to be in a different wrapper- they cannot just be separated with line breaks.  We cannot style them, inject data, anything in this manner.  

2) This isnt using Angular 2- this should be generated from an Array.  This was covered in the tutorial.

3) This doesnt even begin to look like the UX design we had agreed upon.